apps/zotonic_launcher/src/command/zotonic_cmd_addsite.erl

```diff
@@ -189,6 +192,10 @@ parse_args([ "-n", Schema | Args ], Acc) ->
     parse_args(Args, Acc#{ dbschema => Schema });
 parse_args([ "-a", Pw | Args ], Acc) ->
     parse_args(Args, Acc#{ admin_password => Pw });
+parse_args([ "-S", "true" | Args ], Acc) ->
+    parse_args(Args, Acc#{ supervisor => true });
+parse_args([ "-S", "false" | Args ], Acc) ->
```

Comment:
If set `false` by default in place where this value is expected - will no make sense to have set `false` here, because `false` should be value by default.